tests/publiccloud/xfsprepare.pm

```diff
@@ -179,6 +180,8 @@ sub run {
     partition_disk($device, $mnt_xfs, $mnt_scratch);
     create_config($device, $mnt_xfs, $mnt_scratch);
     script_run("source $CONFIG_FILE");
+
+    autotest::loadtest("tests/xfstests/run.pm", run_args => $args);
```

Comment:
```suggestion
    autotest::loadtest("xfstests/run.pm", run_args => $args);
```
I think (not sure) that loadtest does not need in path `test/`

Reply:
It needs "tests/". It was moved here because "xfsprepare" can fail, at which point makes no sense to continue execution of the remaining tests that "run" schedules.